utils/ifupdown.sh.in

```diff
@@ -93,15 +93,36 @@ done
 # We finish setting up the bridge
 if [ "$MODE" = 'start' ]; then
 
+  # check if mstpd is running before bridge-stp may spawn it
+  found=$(pidof -c -s mstpd)
```

Comment:
Should probably be quoted: `found="$(...)"`.  Variable name could also be better ... maybe `mstpdpid`?

Reply:
I'll update the variable name; quoting isn't necessary as word splitting doesn't apply here (eg. in bash or /bin/sh try x="a b"; y=$x; echo $y)... I could still add if you want (doesn't hurt), but then should probably be used in the $("$mstpctl" ...) further down in the same file...

Reply:
Yeah, I'm aiming for consistency, and we quote stuff like that in several other places, so let's do it here too.  And yes, we should probably add quotes on curfwddly (that was added in another PR, and I must have missed it when reviewing that one).

Reply:
OK, with the removal of the loop (sleep in bridge-stp), there's no pidof call anymore, but I'll quote the curfwddly anyway :)